File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Entities/Tenants/TenantAOOModel.cs
using System;

namespace VecompSoftware.DocSuiteWeb.Model.Entities.Tenants
{
    public class TenantAOOModel
    {
        #region [ Constructor ]
        public TenantAOOModel() { }
        #endregion

        #region [ Properties ]
        public Guid IdTenantAOO { get; set; }
        public string Name { get; set; }
        public string Note { get; set; }
        public string CategorySuffix { get; set; }
        public TenantType TenantType { get; set; }
        public string RegistrationUser { get; set; }
        public DateTimeOffset RegistrationDate { get; set; }
        public string LastChangedUser { get; set; }
        public DateTimeOffset? LastChangedDate { get; set; }
        #endregion
    }
}

File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Tenants/TenantAOOModel.ts
import TenantViewModel = require("ViewModels/Tenants/TenantViewModel");
import TenantTypologyTypeEnum = require("TenantTypologyTypeEnum");

interface TenantAOOModel {
    UniqueId: string;
    Name: string;
    Note: string;
    CategorySuffix: string;
    Tenants: TenantViewModel[];
    TenantTypology: TenantTypologyTypeEnum;
}

export = TenantAOOModel;